CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,38 @@
+# The Amethyst Code of Conduct
+
+## Conduct
+
+**Contact**: [amethyst-mods@amethyst-engine.org](mailto:amethyst-mods@amethyst-engine.org)
+
+* We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
+* On Discord, the forums, IRC, GitHub, or any other community area, please avoid using overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
+* Please be kind and courteous. There's no need to be mean or rude.
+* Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+* Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+* We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term "harassment" as including the definition in the <a href="http://citizencodeofconduct.org/">Citizen Code of Conduct</a>; if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
```

Comment:
> In particular, we don't tolerate behavior that excludes people in socially marginalized groups.

Should be:
"In particular, we don't tolerate behavior that excludes people."

Its pretty easy to think about the big group hitting the smaller group, but the opposite is also true.

Reply:
I am not comfortable changing this. We wouldn't tolerate hate groups, for example. One of the core parts of Rust's community is that it, and the community of many Rust projects, pro-actively take steps to protect socially marginalized groups that are historically demonized or threatened online or in real-life.

Reply:
But we wouldn't tolerate hate group because as stated elsewhere we don't tolerate hatefulness. I think we already cover elsewhere the exceptions to that principle.

Reply:
I am, however, ok adding something like "We strive to be an inclusive community"

Reply:
I don't think it is necessary to be so specific. I think it is pretty clear what we are talking about, and so can be as general as it gets.

Reply:
What do you mean?

Reply:
@Moxinilian I'm not sure what you're asking. Can you rephrase?

Reply:
I think jojo's suggestion is legitimate for the reasons I stated above.